cmd/clusterctl/clusterdeployer/clusterclient/clusterclient.go

```diff
@@ -754,43 +754,53 @@ func (c *client) WaitForResourceStatuses() error {
 		klog.V(2).Info("Waiting for Cluster API resources to have statuses...")
 		clusters, err := c.clientSet.ClusterV1alpha1().Clusters("").List(metav1.ListOptions{})
 		if err != nil {
+			klog.V(10).Infof("retrying: failed to list clusters:", err)
 			return false, nil
 		}
 		for _, cluster := range clusters.Items {
 			if reflect.DeepEqual(clusterv1.ClusterStatus{}, cluster.Status) {
+				klog.V(10).Infof("retrying: cluster status is empty")
```

Comment:
Test failures are related to using `Infof` without additional arguments, so this line and others like it should use `Info` instead.
```suggestion
				klog.V(10).Info("retrying: cluster status is empty")
```